src/views/FileBrowserView/PublishFileBrowser.vue

```diff
@@ -278,6 +291,11 @@ export default {
       return publishRest.assetDownloadURI(this.identifier, this.version, asset_id);
     },
 
+    viewURI(asset_id) {
+      return "https://bioimagesuiteweb.github.io/webapp/overlayviewer.html?image=" + publishRest.assetDownloadURI(this.identifier, this.version, asset_id);
+    },
+
+
```

Comment:
This should resolve the current linting errors. `yarn lint` should autofix and show any new errors.
```suggestion
    viewURI(asset_id) {
      return `https://bioimagesuiteweb.github.io/webapp/overlayviewer.html?image=${publishRest.assetDownloadURI(this.identifier, this.version, asset_id)}`;
    },

```